test_runner.py

```diff
@@ -226,8 +226,8 @@ def build_test_list():
         OverrideDefinitions(
             [
                 [
-                    f"--optimizer.name Adam --optimizer.fused --job.dump_folder {args.output_dir}/fused_adamw/",
-                    f"--optimizer.name AdamW --optimizer.fused --job.dump_folder {args.output_dir}/fused_adamw/",
+                    f"--optimizer.name Adam --optimizer.fused --job.dump_folder",
+                    f"--optimizer.name AdamW --optimizer.fused --job.dump_folder",
```

Comment:
I think it's already specified by https://github.com/pytorch/torchtitan/blob/main/test_runner.py#L252
Can we just remove this arg locally, following other tests?